tests/openbis_functions/test_openbis_interbis.py

```diff
@@ -54,25 +62,29 @@ def sample_dict():
 def setup(pytestconfig):
     login_val = pytestconfig.getoption('--login')
     password_val = pytestconfig.getoption('--password')
+    chosen_runner = pytestconfig.getoption('--url')
 
-    o = Interbis(Constants.db_url.value)
+    # o = Interbis(Constants.db_url.value)
+    o = Interbis(chosen_runner, verify_certificates=False)
 
     if login_val != 'no_cl_login' and password_val != 'no_cl_password':
-        o.connect_to_datastore(username=login_val, password=password_val)
+        o.connect_to_datastore(username=login_val, password=password_val, )
```

Comment:
why is there a comma? Is there missing something?

Reply:
deleted